Formula/opencascade.rb

```diff
@@ -0,0 +1,45 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "freeimage"
+  depends_on "tbb"
+  depends_on "gl2ps"
+  depends_on "tcl-tk"
+  depends_on "doxygen"
+
+  def install
+    args = std_cmake_args
+    args << "-DUSE_GL2PS:BOOL=ON"
+    args << "-DUSE_FREEIMAGE:BOOL=ON"
+    args << "-DUSE_TBB:BOOL=ON"
+    args << "-DINSTALL_DOC_Overview:BOOL=ON"
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make", "install"
+    end
+
+    # Some apps expect resources in legacy ${CASROOT}/src directory
+    cd prefix do
+      ln_s "share/opencascade/resources", "src"
+    end
+  end
+
+  def caveats; <<~EOS
+    Some apps will require this enviroment variable:
+      CASROOT=#{opt_prefix}
+    EOS
+  end
+
+  test do
+    ENV["CASROOT"] = opt_prefix
+    assert_match "1", shell_output("#{bin}/DRAWEXE -c \"pload ALL\"")
```

Comment:
can we use assert_equal?

Reply:
@fxcoudert recommended `assert_match` in an earlier because the legacy formula didn't even use `assert`.  What is the benefit of `assert_equal` over `assert_match`?

Reply:
`assert_match` is for when we can only easily match a regex or a substring. `assert_equal` is more precise and sometimes requires just a `.chomp` on the end of the `shell_output`

Reply:
done